crypto/src/main/scala/org/bitcoins/crypto/CryptoBytesUtil.scala

```diff
@@ -63,17 +66,22 @@ trait CryptoBytesUtil {
   /** Flips the endianness of the given sequence of bytes. */
   def flipEndianness(bytes: ByteVector): String = encodeHex(bytes.reverse)
 
+  private val Z: Char = '0'
   /**
     * Adds the amount padding bytes needed to fix the size of the hex string
     * for instance, ints are required to be 4 bytes. If the number is just 1
     * it will only take 1 byte. We need to pad the byte with an extra 3 bytes so the result is
     * 00000001 instead of just 1.
     */
-  private def addPadding(charactersNeeded: Int, hex: String): String = {
-    val paddingNeeded = charactersNeeded - hex.length
-    val padding = Vector.fill(paddingNeeded)("0")
-    val paddedHex = padding.mkString + hex
-    paddedHex
+  final def addPadding(charactersNeeded: Int, hex: String): String = {
```

Comment:
nit: `charactersNeeded -> paddingNeeded` as it is no longer how many total characters are needed